contrib/check-config.sh

```diff
@@ -191,20 +200,23 @@ if [ "$(cat /sys/module/apparmor/parameters/enabled 2> /dev/null)" = 'Y' ]; then
 	fi
 fi
 
-flags=(
-	NAMESPACES {NET,PID,IPC,UTS}_NS
-	CGROUPS CGROUP_CPUACCT CGROUP_DEVICE CGROUP_FREEZER CGROUP_SCHED CPUSETS MEMCG
-	KEYS
-	VETH BRIDGE BRIDGE_NETFILTER
-	IP_NF_FILTER IP_NF_TARGET_MASQUERADE
-	NETFILTER_XT_MATCH_{ADDRTYPE,CONNTRACK,IPVS}
-	NETFILTER_XT_MARK
-	IP_NF_NAT NF_NAT
-
-	# required for bind-mounting /dev/mqueue into containers
+check_flags \
+	NAMESPACES NET_NS \
+	NAMESPACES PID_NS \
+	NAMESPACES IPC_NS \
+	NAMESPACES UTS_NS \
```

Comment:
Uh, this isn't quite right :sweat_smile: (checking `NAMESPACES` four times :muscle:)

```suggestion
	NAMESPACES NET_NS PID_NS IPC_NS UTS_NS \
```

Reply:
Oh my, what a mistake from my side! A typical example of when the human brain tricks us to see what we believe we see; underscores are not spaces and the other way around. :-) Thanks for spotting!